ajax/libs/intl-messageformat/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "intl-messageformat",
+  "filename": "intl-messageformat.min.js",
+  "version": "2.2.0",
+  "description": "Formats ICU Message strings with number, date, plural, and select placeholders to create localized messages.",
+  "keywords": [
+    "i18n",
+    "intl",
+    "internationalization",
+    "localization",
+    "globalization",
+    "messageformat",
+    "parser",
+    "plural",
+    "icu"
+  ],
+  "author": "Eric Ferraiuolo <eferraiuolo@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:yahoo/intl-messageformat.git"
```

Comment:
Let's use `git://github.com/yahoo/intl-messageformat.git`
This can clone the repo without SSL key.
Thanks